src/wasmtime/crates/wasmtime/src/runtime/store.rs

```diff
@@ -1115,6 +1115,11 @@ impl<T> Store<T> {
         self.inner.set_epoch_deadline(ticks_beyond_current);
     }
 
+    // get current epoch deadline
```

Comment:
This comment seems to just restate the function name in comment form,  is there more information we can provide here?